src/DeleteModal.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react'
+import Button from 'react-bootstrap/lib/Button'
+
+function DeleteModal(props) {
+  function confirmDelete() {
+    props.shouldDelete(true);
+  }
+  function rejectDelete() {
+    props.shouldDelete(false);
+  }
+  return (
+    <div>
+      <h1>Are you sure?</h1>
+      <Button onClick={confirmDelete} name="Yes">Yes</Button>
+      <Button onClick={rejectDelete} name="No">No</Button>
+    </div>
+  );
+}
+
+DeleteModal.propTypes = {
+  shouldDelete: PropTypes.func.isRequired,
+};
+
+export default DeleteModal;
```

Comment:
There is an issue about this buried in the backlog. I would pull this out of this PR and make a new one for that.